pkg/client/complete-service-examples-testfile.json

```diff
@@ -0,0 +1,25 @@
+[
```

Comment:
test fixtures should be put under a `testdata/` directory in the same package as the test--this should be `pkg/client/testdata/complete-service-examples-testfile.json`

It's a very undocumented feature of go: https://medium.com/@povilasve/go-advanced-tips-tricks-a872503ac859

Reply:
Thanks for letting me know!

Reply:
Thanks for pointing that out! File moved.